tests/ddl_reentrant/run.sh

```diff
@@ -103,7 +103,7 @@ function ddl_test() {
 
     echo $ddl > ${WORK_DIR}/ddl_temp.sql
     ensure 10 check_ddl_executed "${WORK_DIR}/cdc.log" "${WORK_DIR}/ddl_temp.sql" true
-    ddl_start_ts=$(grep "Execute DDL succeeded" ${WORK_DIR}/cdc.log|tail -n 1|grep -oE '"start_ts\\":[0-9]{18}'|awk -F: '{print $(NF)}')
+    ddl_start_ts=$(grep "Execute DDL succeeded" ${WORK_DIR}/cdc.log|tail -n 1|grep -oE '"StartTs\\":[0-9]{18}'|awk -F: '{print $(NF)}')
```

Comment:
Why change the keyword from `start_ts` to `StartTs`

Reply:
because the log changed

Reply:
https://github.com/pingcap/ticdc/blob/master/cdc/changefeed.go#L741 in the old owner
https://github.com/pingcap/ticdc/blob/master/cdc/owner/async_sink.go#L137 in the new owner